test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityControllerSpec.scala

```diff
@@ -852,6 +923,67 @@ class YearToDateLiabilityControllerSpec
 
       }
 
+      "handling a TRUST user on a non-calculated journey" must {
+
+        "display the page" when {
+
+          def testCompletedWithTrust(
+            answers: YearToDateLiabilityAnswers,
+            backLink: Call,
+            extraChecks: Document => Unit = _ => ()
+          ): Unit = {
+            val (session, fillingOutReturn, _) = sessionWithMultipleDisposalsState(answers)
+            val fillingOutReturnWithTrust = fillingOutReturn.copy(subscribedDetails =
+              fillingOutReturn.subscribedDetails.copy(name = sample[TrustName].asLeft[IndividualName])
+            )
+            val sessionDataWithTrust = session.copy(journeyStatus = Some(fillingOutReturnWithTrust))
+
+            inSequence {
+              mockAuthWithNoRetrievals()
+              mockGetSession(sessionDataWithTrust)
+            }
+
+            checkPageIsDisplayed(
+              performAction(),
+              messageFromMessageKey("hasEstimatedDetails.title"), { doc =>
```

Comment:
is there no trust specific content you can assert on?

Reply:
Yes. Done.

Reply:
Done.